task_DEV_4/task_DEV_4/Square.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace task_DEV_4
+{
+    class Square
+    {
+        public double Side { get; set; }
```

Comment:
Maybe it would be better to rename this property like firstSide? To have more convinient naming for inheritance. Please think how to name it better.

Reply:
Done